actors/miner/src/lib.rs

```diff
@@ -216,6 +223,66 @@ impl Actor {
         })
     }
 
+    /// Returns the owner address
+    fn get_owner(rt: &mut impl Runtime) -> Result<GetOwnerReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let state: State = rt.state()?;
+        let owner = get_miner_info(rt.store(), &state)?.owner;
+        Ok(GetOwnerReturn { owner })
+    }
+
+    /// Returns the worker address
+    fn is_controlling_address(
+        rt: &mut impl Runtime,
+        params: IsControllingAddressParam,
+    ) -> Result<IsControllingAddressReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let state: State = rt.state()?;
```

Comment:
Resolve the parameter address to an ID address, as all the ones in state are.

Reply:
Could also just use ActorID here? There's no meaningful invocation of this with an unresolved address?

Reply:
I think it's just possible enough to use an address (e.g. of a smart wallet) of a possibly-not-deployed actor here